examples/playbooks/become.transformed.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
[This](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/become.transformed.yml) is a reformatted copy of [become.yml](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/become.yml)